pkg/cmd/api/api.go

```diff
@@ -85,6 +85,9 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			any value that contains %[1]s{...}%[1]s in quotes to prevent the shell from
 			applying special meaning to curly braces.
 
+			This command supports invoking preview-stage API endpoints/features, when
```

Comment:
```suggestion
			This command supports opting into previews, which are feature-flagged, experimental API endpoints or behaviors. The API expects opt-in via the %[1]sAccept%[1]s header with format %[1]sapplication/vnd.github.<preview-name>-preview+json%[1]s and this command facilitates that via %[1]s--preview <preview-name>%[1]s.
```

What do you think of something like this? Happy for you to adjust a little, just trying to provide a little more detail to the purpose and use of this thing, since the `api` command is such a thin wrapper over the real request/responses.

Reply:
I wrote a longer response on my laptop but ran out of time.

I asked for the details of `--preview` to be added in my issue but the staff person pushed back against it. 

So I'm definitely happy to accept this proposal as it aligns with my original request. 

If you're still ok with it, I'll pull it in when I get back to a computer.

Reply:
I think it's hard to explain the intention of `--preview` without a bit of the underlying details, and I think with the intention of `gh api` being a thin layer over the raw requests it's not too much. If there were a GitHub docs link that explained this I would just link to that but like you, I couldn't find one that wasn't just like: "yeah here's some previews that graduated" with no real explanation. I think these are rarely used, and perhaps have become less used since the `api` command was first introduced.

Reply:
Right. If I had a doc like that, I'd be half tempted to include it in the documentation...

Anyway, I'll fold in these suggestions and push. I'm quite happy with this as it aligns with my initial request.

Reply:
If you guys think it's best to include the underlying detail, I'm sure down for it.